### PR TITLE
Print a WARN message when the Management Api is enabled

### LINKED ISSUE #170: Print a WARN message when the Management API is enabled
**Description:**

Need to print a log message in the micro integrator startup when the Management API is enabled 

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by SanojPunchihewa**
## Purpose
With this PR  a WARN message is printed if the Management Api is enabled. Fixes #170

## Goals
Following WARN message is logged with this fix.
`[micro-integrator]  WARN - ManagementInternalAPI Micro Integrator Management REST API is enabled`

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/ManagementInternalAPI.java [hunk 1]
```diff
@@ -35,6 +37,12 @@
 
     private String name;
 
+    private static Log log = LogFactory.getLog(ManagementInternalAPI.class);
```
Comment (a5anka) [inline components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/ManagementInternalAPI.java]: This constant name does not match the WSO2 code style. Shall we fix?

Comment (SanojPunchihewa) [reply components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/ManagementInternalAPI.java]: Fixed in 2fcc0a2

File: components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/ManagementInternalApi.java [hunk 2]
```diff
@@ -32,10 +34,16 @@
...
 
-public class ManagementInternalAPI implements InternalAPI {
+public class ManagementInternalApi implements InternalAPI {
 
     private String name;
 
+    private static Log LOG = LogFactory.getLog(ManagementInternalApi.class);
+
+    public ManagementInternalApi(){
+        LOG.warn("Management Api has been enabled");
```
Comment (a5anka) [inline components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/ManagementInternalApi.java]:
```suggestion
        LOG.warn("Micro Integrator Management REST API is enabled");
```

Comment (SanojPunchihewa) [reply components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/ManagementInternalApi.java]: Fixed in c5a0950

**Pull request merged**